trinity/main.py

```diff
@@ -192,6 +192,13 @@ def main() -> None:
         logger.info('Networking process (pid=%d) terminated', networking_process.pid)
 
 
+def run_console(chain_config: ChainConfig, vanilla_shell_args: bool) -> None:
+    try:
+        console(chain_config.jsonrpc_ipc_path, use_ipython=vanilla_shell_args)
+    except FileNotFoundError:
+        sys.exit(1)
```

Comment:
I think this is missing a logging statement?  Maybe I'm dense but I don't see anywhere where the error will make it's way to the console.

Reply:
@pipermerriam yes, that's right. Seems it got lost in that refactoring. 

@glaksmono 
The message is only inside the exception now but it still needs to be printed to the user.

Reply:
My bad. Just fixed it @pipermerriam @cburgdorf